src/zope/app/file/browser/file.py

```diff
@@ -293,7 +305,7 @@ def update_object(self, data, contenttype):
         # Update *only* if a new value is specified
         if data:
             self.context.data = data
-            descriptor.attributes += "data",
+            descriptor.attributes += ("data",)
```

Comment:
pylint recently got smart enough to complain that a trailing comma with no parenthesis can be hard/confusing to read and is as likely to be an accident as it is to be correct (and I have seen that in the wild).